src/yadism/esf/exs.py

```diff
@@ -5,7 +5,67 @@
 from .esf import ESFInfo
 from .result import ESFResult, EXSResult
 
-conv = 3.893793e10  # conversion factor from GeV^-2 to 10^-38 cm^2
+GEV_CM2_CONV = 3.893793e10
+"Conversion factor from GeV^-2 to 10^-38 cm^2"
+
+
+def xs_coeffs(kind, y, x=None, Q2=None, params=None):
+    """Compute coefficients in the definition of a given cross section.
+
+    Parameters
+    ----------
+    kind : str
+        the identifier of the cross section for which coefficients have to be
+        computed
+    y : float
+    x: None or float
+    Q2: None or float
+    params : None or dict
+
+    """
+    yp = 1.0 + (1.0 - y) ** 2
+    ym = 1.0 - (1.0 - y) ** 2
+    yL = y**2
+
+    # if kind == "XSreduced":
```

Comment:
Let's remove this code which we thought once upon a time we would maybe need - we accept the thing changes dataset by dataset

Reply:
Ok, I'll do.

Still thinking to improve the names, but for a *long* time being it is just fine.